### ISSUE TITLE
aria-live console output repeats what has been announced in Chrome

### ISSUE THREAD

**Issue by jooyoungseo** [rstudio/rstudio#6291]
### System details

RStudio Edition : Server
RStudio Version : 1.3.845
OS Version      : Windows Chrome with NVDA 2019.3.1/JAWS 2020.2001.70.400
R Version       : any

### Steps to reproduce the problem

1. Load RStudio Server in Chrome.
2. Enable accessibility mode.
3. Switch to "Focus Mode" for NVDA by pressing NVDA+Spacebar; for JAWS, turn off virtual cursor to fully activate "Forms Mode" by pressing JAWS+Z.
4. Focus in "Console" by pressing CTRL+2.
5. Clear console output by pressing CTRL+L.
6. Type in the following test code.

``` r
chooseCRANmirror(ind=1)  # Press enter.
install.packages(’tidyverse’)  # Press enter.
```

### Describe the problem in detail

aria-live console output keeps repeating what it has already read for both NVDA and JAWS in Chrome; FireFox does not have this issue.

Interestingly, this repeat has its own pattern: it adds a new content at the end while reading all the way to the top each cycle. This pattern can be found with NVDA and JAWS in Chrome.

The following is what both screen readers speak in response to the command invoked above:

```
also installing the dependencies ‘rlang’, ‘tidyselect’, ‘broom’, ‘dbplyr’, ‘dplyr’, ‘haven’, ‘modelr’, ‘rvest’, ‘tidyr’, ‘xml2’

trying URL ’https://cloud.r-project.org/src/contrib/rlang_0.4.4.tar.gz’
> install.packages(’tidyverse’)
also installing the dependencies ‘rlang’, ‘tidyselect’, ‘broom’, ‘dbplyr’, ‘dplyr’, ‘haven’, ‘modelr’, ‘rvest’, ‘tidyr’, ‘xml2’

trying URL ’https://cloud.r-project.org/src/contrib/rlang_0.4.4.tar.gz’
Content type ’application/x-gzip’ length 813413 bytes (794 KB)
==================================================
downloaded 794 KB

trying URL ’https://cloud.r-project.org/src/contrib/tidyselect_1.0.0.tar.gz’
> install.packages(’tidyverse’)
also installing the dependencies ‘rlang’, ‘tidyselect’, ‘broom’, ‘dbplyr’, ‘dplyr’, ‘haven’, ‘modelr’, ‘rvest’, ‘tidyr’, ‘xml2’

trying URL ’https://cloud.r-project.org/src/contrib/rlang_0.4.4.tar.gz’
Content type ’application/x-gzip’ length 813413 bytes (794 KB)
==================================================
downloaded 794 KB

trying URL ’https://cloud.r-project.org/src/contrib/tidyselect_1.0.0.tar.gz’
Content type ’application/x-gzip’ length 151161 bytes (147 KB)
==================================================
downloaded 147 KB

trying URL ’https://cloud.r-project.org/src/contrib/broom_0.5.4.tar.gz’
> install.packages(’tidyverse’)
also installing the dependencies ‘rlang’, ‘tidyselect’, ‘broom’, ‘dbplyr’, ‘dplyr’, ‘haven’, ‘modelr’, ‘rvest’, ‘tidyr’, ‘xml2’

trying URL ’https://cloud.r-project.org/src/contrib/rlang_0.4.4.tar.gz’
Content type ’application/x-gzip’ length 813413 bytes (794 KB)
==================================================
downloaded 794 KB

trying URL ’https://cloud.r-project.org/src/contrib/tidyselect_1.0.0.tar.gz’
Content type ’application/x-gzip’ length 151161 bytes (147 KB)
==================================================
downloaded 147 KB

trying URL ’https://cloud.r-project.org/src/contrib/broom_0.5.4.tar.gz’
Content type ’application/x-gzip’ length 1228129 bytes (1.2 MB)
==================================================
downloaded 1.2 MB

trying URL ’https://cloud.r-project.org/src/contrib/dbplyr_1.4.2.tar.gz’
> install.packages(’tidyverse’)
also installing the dependencies ‘rlang’, ‘tidyselect’, ‘broom’, ‘dbplyr’, ‘dplyr’, ‘haven’, ‘modelr’, ‘rvest’, ‘tidyr’, ‘xml2’

trying URL ’https://cloud.r-project.org/src/contrib/rlang_0.4.4.tar.gz’
Content type ’application/x-gzip’ length 813413 bytes (794 KB)
==================================================
downloaded 794 KB

trying URL ’https://cloud.r-project.org/src/contrib/tidyselect_1.0.0.tar.gz’
Content type ’application/x-gzip’ length 151161 bytes (147 KB)
==================================================
downloaded 147 KB

trying URL ’https://cloud.r-project.org/src/contrib/broom_0.5.4.tar.gz’
Content type ’application/x-gzip’ length 1228129 bytes (1.2 MB)
==================================================
downloaded 1.2 MB

trying URL ’https://cloud.r-project.org/src/contrib/dbplyr_1.4.2.tar.gz’
Content type ’application/x-gzip’ length 303397 bytes (296 KB)
==================================================
downloaded 296 KB

trying URL ’https://cloud.r-project.org/src/contrib/dplyr_0.8.4.tar.gz’
Content type ’application/x-gzip’ length 1179051 bytes (1.1 MB)
Too much console output to announce.
```

### Describe the behavior you expected

aria-live console output must be read one time only for new content.

**Comment by jooyoungseo** [comment]
I do not have a permission to label this issue for @gtritchie. Please take care of this!
